### PR TITLE
PR for Cart and Wishlist Pages

### PR DESCRIPTION
**Pull request by manojasarsa**
This PR includes :
- Add to Cart: It adds the item to the cart & once added shows "Go to Cart" on the product card.
- Remove the product from the cart
- Add to Wishlist: On click, it adds the item in the wishlist & vice-verse.
- Remove the item from the wishlist
- Increase or Decrease the quantity of a particular product.
- If the cart already contains that item, it's only increasing the quantity.

### CODE REVIEW

File: src/components/PriceBox/PriceBox.jsx [hunk 1]
```diff
@@ -1,6 +1,20 @@
...
+    const cartCounter = cartState.cartItems.length;
+    const finalAmount = totalPrice.tPrice - totalPrice.dPrice + 99;
+    const savedAmount = totalPrice.tPrice - finalAmount;
+
+    // const {_id, price, discountedPrice, qty } = product;
+
+    console.log("totalPrice:", totalPrice);
+    // console.log("discounted totalPrice in Cart:", totalDiscountedPrice);
+    console.log("finalamount:", finalAmount);
+
```
Comment (samyakshah3008) [inline src/components/PriceBox/PriceBox.jsx]: You can later on remove this commented console logs

**Review by samyakshah3008 — APPROVED** [review]
Clean code 
Naming conventions are self explanatory 
Suggestions given as single comment 

Rest is all good for approval ✅

**Comment by samyakshah3008** [comment]
Loved the PR description, made easy for better code review ✅

File: src/components/PriceBox/PriceBox.jsx (hunk 1 above)
Comment (manojasarsa) [reply src/components/PriceBox/PriceBox.jsx]: done!

**Review by manojkumar8400 — APPROVED** [review]
good to go.

**Review by mimansha-swarup — APPROVED** [review]
code looks clean good to merge

**Pull request merged**